satellite/audit/disqualification_test.go

```diff
@@ -45,9 +45,6 @@ func TestDisqualificationTooManyFailedAudits(t *testing.T) {
 			},
 		},
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		err := planet.Satellites[0].Audit.Service.Close()
```

Comment:
There are several places in this file where you removed `satellitePeer.Audit.Service.Close()` and did not replace it with `satellitePeer.Audit.Worker.Close()`.
On that note, `worker.Close()` doesn't currently do anything, but it should do `worker.Loop.Close()`

Reply:
Closing the audit worker is important still because these tests will be affected by passing/failing audits that might run in the background if the worker isn't stopped.

Reply:
ah you're right, I definitely forgot to add it back.

Reply:
hm I'm getting a panic: close on closed channel now that I've added `worker.Loop.Close()` but I'm not totally sure why the channel would be closed already.